app/assets/stylesheets/application.css

```diff
@@ -10,6 +10,5 @@
  *
  *= require_self
  *= require select2
- *= require jquery.ui.datepicker
```

Comment:
don’t know!
Is that a change I made?

Bob Bannister
+44 7977 500360




> On 3 Jun 2016, at 17:09, Owen Bannister <notifications@github.com> wrote:
> 
> In app/assets/stylesheets/application.css <https://github.com/bannio/mars/pull/4#discussion_r65733634>:
> 
> > @@ -10,6 +10,5 @@
> >   *
> >   *= require_self
> >   *= require select2
> > - *= require jquery.ui.datepicker
> Are we not using the datepicker or is it now included elsewhere?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/bannio/mars/pull/4/files/b6897336332661089e43cb007e6147a71de7b7f6#r65733634>, or mute the thread <https://github.com/notifications/unsubscribe/AE50A6dC0Jx-eKMxG9bcV6CbIegSlMjRks5qIFG9gaJpZM4IlQlN>.
>